From b4494c830a0ea45800ad7aa03543b965cfc17f46 Mon Sep 17 00:00:00 2001
From: Charles Keepax <ckeepax@opensource.wolfsonmicro.com>
Date: Thu, 13 Feb 2014 16:05:20 +0000
Subject: [PATCH 131/351] mfd: florida: Add defaults for the charge pump
 settings

Ensure these are synchronised, if we have updated then due to the
headphone detection.

Change-Id: I9cb6dc7afd0b9a45f517ec78b447ea4d9bcc262c
Signed-off-by: Charles Keepax <ckeepax@opensource.wolfsonmicro.com>
Signed-off-by: Nikesh Oswal <Nikesh.Oswal@wolfsonmicro.com>
---
 drivers/mfd/florida-tables.c |   32 ++++++++++++++++++++++++++++++++
 1 file changed, 32 insertions(+)

diff --git a/drivers/mfd/florida-tables.c b/drivers/mfd/florida-tables.c
index cf5db8a..b7c4ffe 100644
--- a/drivers/mfd/florida-tables.c
+++ b/drivers/mfd/florida-tables.c
@@ -633,8 +633,40 @@ static const struct reg_default florida_reg_default[] = {
 	{ 0x00000440, 0x8FFF },    /* R1088  - DRE Enable */
 	{ 0x00000450, 0x0000 },    /* R1104  - DAC AEC Control 1 */
 	{ 0x00000458, 0x0000 },    /* R1112  - Noise Gate Control */
+	{ 0x00000460, 0x0C40 },
+	{ 0x00000461, 0x8000 },
+	{ 0x00000462, 0x0C41 },
+	{ 0x00000463, 0x4820 },
+	{ 0x00000464, 0x0C41 },
+	{ 0x00000465, 0x4040 },
+	{ 0x00000466, 0x0C41 },
+	{ 0x00000467, 0x3940 },
+	{ 0x00000468, 0x0C42 },
+	{ 0x00000469, 0x2030 },
+	{ 0x0000046A, 0x0842 },
+	{ 0x0000046B, 0x1990 },
+	{ 0x0000046C, 0x08C2 },
+	{ 0x0000046D, 0x1450 },
+	{ 0x0000046E, 0x08C6 },
+	{ 0x0000046F, 0x1020 },
+	{ 0x00000470, 0x08C6 },
+	{ 0x00000471, 0x0CD0 },
+	{ 0x00000472, 0x08C6 },
+	{ 0x00000473, 0x0A30 },
+	{ 0x00000474, 0x0442 },
+	{ 0x00000475, 0x0660 },
+	{ 0x00000476, 0x0446 },
+	{ 0x00000477, 0x0510 },
+	{ 0x00000478, 0x04C6 },
+	{ 0x00000479, 0x0400 },
+	{ 0x0000047A, 0x04CE },
+	{ 0x0000047B, 0x0330 },
+	{ 0x0000047C, 0x05DF },
+	{ 0x0000047D, 0x0001 },
+	{ 0x0000047E, 0x07FF },
 	{ 0x00000480, 0x0040 },    /* R1152  - Class W ANC Threshold 1 */
 	{ 0x00000481, 0x0040 },    /* R1153  - Class W ANC Threshold 2 */
+	{ 0x00000483, 0x0826 },
 	{ 0x00000490, 0x0069 },    /* R1168  - PDM SPK1 CTRL 1 */
 	{ 0x00000491, 0x0000 },    /* R1169  - PDM SPK1 CTRL 2 */
 	{ 0x00000492, 0x0069 },    /* R1170  - PDM SPK2 CTRL 1 */
-- 
1.7.9.5

